cpp/modmesh/toggle/RadixTree.cpp

```diff
@@ -72,6 +72,132 @@ void CallProfiler::print_profiling_result(const RadixTreeNode<CallerProfile> & n
     }
 }
 
+void CallProfilerSerializer::serialize_call_profiler(const CallProfiler & profiler, std::ostream & outstream)
+{
+    // Example:
```

Comment:
We do not need to keep examples everywhere.  The example in this block of comments seems to duplicate with that in google test.  Please keep the code here clean by removing the unnecessary comments.

And marking only "Example" in this block does not help a maintainer to read.  The only common information between the code and comment in the function is `"radix_tree"`.  It does not help maintainability.

If anything in the example comments here should be kept, please make it concise.  And try to use doxygen format if it fits.